### PR TITLE
Neo4j nodes with count metadata option implemented

### PR DESCRIPTION
**Pull request by dliubimov**

### CODE REVIEW

**Review by ruscooler — APPROVED** [review]

File: odd_neo4j_adapter/mappers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from collections import namedtuple
+
+_match_all_nodes: str = 'MATCH (n) RETURN distinct labels(n), count(*)'
```
Comment (DementevNikita) [inline odd_neo4j_adapter/mappers/__init__.py:3]: Is it the query you gathering nodes with? If so, can we remove that from `mappers` package? It seems off to me to keep that here tbh

File: odd_neo4j_adapter/cache.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                            data_transformer_runs: Iterable[DataEntity],
+                            updated_at: datetime = datetime.now()):
+        self.__DATA_ENTITIES = list(chain(datasets, data_transformers, data_transformer_runs)), updated_at
+
+    def retrieve_data_entities(self, changed_since: datetime = None) -> Optional[CacheEntry]:
+        if self.__DATA_ENTITIES is None:
+            return None
+
+        data_entities_filtered = self.__DATA_ENTITIES[0]
+        # data_entities_filtered = [
```
Comment (DementevNikita) [inline odd_neo4j_adapter/cache.py:26]: Can you remove the commented section?

File: odd_neo4j_adapter/adapter.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        return []
+
+    def __query(self, tx, cyp: str) -> list:
+        return tx.run(cyp).values()
+
+    def __execute(self, cyp: str) -> list[tuple]:
+        with self.__connection.session() as session:
+            return session.read_transaction(self.__query, cyp)
+
+    # replace
```
Comment (DementevNikita) [inline odd_neo4j_adapter/adapter.py:55]: What for?

**Review by DementevNikita — CHANGES_REQUESTED** [review]
Comments

**Review by DementevNikita — APPROVED** [review]

**Pull request merged**